src/components/Popover/README.md

```diff
@@ -418,7 +424,7 @@ See Apple’s Human Interface Guidelines and API documentation about accessibili
 
 Popovers usually contain an [option list](https://polaris.shopify.com/components/lists-and-tables/option-list) or an [action list](https://polaris.shopify.com/components/actions/action-list), but can also contain other controls or content.
 
-A popover can contain many numerous types of content. Whether it's a menu, grid or something entirely different! When `aria-expanded` is applied to an element, `aria-haspopup` will default to `menu`. To assist screen readers you'll need to pass `ariaHaspopup` to `Popover`.
+A popover can contain many numerous types of content. Whether it's a menu, grid or something entirely different! When `aria-expanded` is applied to an element, `aria-haspopup` will default to `menu`. To assist screen readers with an [action list](https://polaris.shopify.com/components/actions/action-list) you'll need to ensure that the first focusable node is automatically focused on open by passing `autofocusTarget='first-node'` to `Popover`.
```

Comment:
Indeed, I'm discouraging the use of `ariaHaspopup`.  According to ARIA documentation, when a button has the `aria-haspopup` attribute set, screen reader are expecting the content to be a sibling in the DOM tree, but in our case since we are using the Portal, the content is not a sibling.

I've also noticed that the following sentence  is not true since the code will only set the `aria-haspopup` to the value passed as props to the activator.
> When `aria-expanded` is applied to an element, `aria-haspopup` will default to `menu`.


I will re-think on how to rephrase this whole paragraph.